lib/workers/repository/init/vulnerability.ts

```diff
@@ -185,7 +185,10 @@ export async function detectVulnerabilityAlerts(
         };
 
         let matchCurrentVersion = `< ${val.firstPatchedVersion}`;
-        if (datasource === MavenDatasource.id) {
+        if (
+          datasource === MavenDatasource.id ||
+          datasource === NugetDatasource.id
+        ) {
           matchCurrentVersion = `(,${val.firstPatchedVersion})`;
```

Comment:
are you sure it's correct range for nuget? 🤔

Reply:
I've confirmed that our nuget tests include similar ranges (e.g. `(,1.0.0)`)